packages/react-graphql/package.json

```diff
@@ -27,10 +27,12 @@
     "@shopify/async": "^1.1.0",
     "@shopify/react-async": "^1.2.2",
     "@shopify/useful-types": "^1.1.1",
-    "graphql-typed": "^0.3.0",
+    "apollo-client": "^2.5.1",
```

Comment:
I am adding `apollo-client@^2.5.1` to the dependencies list because it is a peer dependency for `react-apollo`. We could alternatively shuffle this dep to `peerDependencies`, but i think it makes more sense as a direct dependency?

Reply:
Hmm yeah I guess this is fine since react-apollo is a regular dependency, but looser version restriction would be 💯 if possible